src/gt4py/next/iterator/transforms/pass_manager.py

```diff
@@ -160,6 +161,10 @@ def apply_common_transforms(
         ir = CommonSubexpressionElimination().visit(ir)
         ir = MergeLet().visit(ir)
 
-    ir = InlineLambdas.apply(ir, opcount_preserving=True)
+    ir = InlineLambdas.apply(
+        ir,
+        opcount_preserving=True,
+        force_inline_lambda_args=(lift_mode == LiftMode.FORCE_INLINE) and force_inline_lambda_args,
```

Comment:
I thought that we might want to inline lambda args only when lift_mode == FORCE_INLINE, because this was the case where I encountered my problem. I can remove this condition.